### PR TITLE
sample_data tests and functions

### PR DESCRIPTION
**Pull request by moniquewong**
Adding finished function, #18 
- Used helper functions like in missing data function

Need to:
- Write tests for this function

### CODE REVIEW

**Comment by moniquewong** [comment]
Next commit will address adding functions that check summarize_data helper function

**Comment by moniquewong** [comment]
Note that at this point some tests are failing right now - currently debugging

**Comment by mirohu** [comment]
Hi Monique! WELL DONE! The functions, the output, and the tests all look great. You did such a beautiful job and I don't know what I can add. Should've added the review before you debugged all the test 👍

File: tests/test_sample_data.py [hunk 1]
```diff
@@ -43,67 +49,57 @@ def test_other_input(make_files):
     with pytest.raises(ValueError):
         sample.sample_data(str(make_files.join('df.pkl')))
 
-def test_sample_record(make_files):
-    path_to_file = str(make_files.join('df.csv'))
-    df = pd.read_csv(path_to_file)
+def test_sample_record():
```
Comment (mirohu) [inline tests/test_sample_data.py:58]: Don't forget to add 1/2 line saying what the tests are doing!

**Review by mirohu — CHANGES_REQUESTED** [review]

File: tests/test_sample_data.py (hunk 1 above)
Comment (moniquewong) [reply tests/test_sample_data.py:58]: Made changes in new commit!

**Pull request merged**